Automattic/simplenote-macos
# Publish Popover Style

### Details:
- Simplenote's Publish Popover will now match Simplenote's Appearance
- Drops imperative **Refresh Publish Popover** invocations. Replaced by **EntityObserver** 🔥

@emilylaguna Em!! you game for a quick PR?

**Note:** This entire UI is scheduled for massive changes (this is a midway step!) 😃

Ref. #458 

### Test: Appearance
1. Click the **Share** action (located above the Editor): <img width="14" alt="UP" src="https://user-images.githubusercontent.com/1195260/86270163-2f8b2b80-bba1-11ea-9e2c-70739c9bb09e.png">
2. Click over **Publish to the Web**
3. While the Share Popover is onScreen click on the System Bar: **View > Theme** (and switch to another theme)

- [ ] Verify the Publish Popover immediately adapts to the new Appearance

### Test: Regressions
1. Click the **Share** action (located above the Editor): <img width="14" alt="UP" src="https://user-images.githubusercontent.com/1195260/86270163-2f8b2b80-bba1-11ea-9e2c-70739c9bb09e.png">
2. Click over **Publish to the Web**

- [ ] Verify that clicking **Publish** displays **publishing...** while the OP is being performed
- [ ] Verify that shortly after, the note gets published, and the URL shows up in the text field
- [ ] Verify that clicking **Unpublish** displays **unpublishing...**
- [ ] Verify that shorly after, you're back into the Unpublished state
### Release
These changes do not require release notes.
